content/getting-started/index.mdx

```diff
@@ -13,7 +13,7 @@ If you're wondering whether LiveChat would suit your business, this is the place
 
 # What LiveChat is
 
-**LiveChat** helps you handle conversations with the customers who visit your websites. Customers can start chats with your agents, but it can also be the other way round. Your agents can initiate chats, making your visitors feel welcome and valued. If you're up to some automation, you can engage them in a conversation with [targeted messages](https://www.livechat.com/help/targeted-messages/). Also, delegating some work to [bots](https://www.livechat.com/help/chatbots-explained/) is an option when running a bussines on a bigger scale.
+**LiveChat** helps you handle conversations with the customers who visit your websites. Customers can start chats with your agents, but it can also be the other way round. Your agents can initiate chats, making your visitors feel welcome and valued. If you're up to some automation, you can engage them in a conversation with [targeted messages](https://www.livechat.com/help/targeted-messages/). Also, delegating some work to [bots](https://www.livechat.com/help/chatbots-explained/) is an option when running a busines on a bigger scale.
```

Comment:
```suggestion
**LiveChat** helps you handle conversations with the customers who visit your websites. Customers can start chats with your agents, but it can also be the other way round. Your agents can initiate chats, making your visitors feel welcome and valued. If you're up to some automation, you can engage them in a conversation with [targeted messages](https://www.livechat.com/help/targeted-messages/). Also, delegating some work to [bots](https://www.livechat.com/help/chatbots-explained/) is an option when running a business on a bigger scale.
```